src/ModelParser/JMSParser.php

```diff
@@ -70,6 +71,10 @@ abstract class BaseJMSParser implements ModelParserInterface
 
     public function __construct(Reader $annotationsReader)
     {
+        if (PHP_VERSION_ID >= 80000) {
+            $annotationsReader = new AttributeReader($annotationsReader);
```

Comment:
this is unrelated the the collection change. does it currently add anything, or would we need to do further steps to actually process annotations?

i really like handling attributes, but we should probably do that in a separate MR and add some test cases for the JMSParser to make sure it works as expected.

Reply:
You are right, I'll move this to a separate PR to keep things separate and also add some tests for this.

Reply:
Implemented via #30